wicket-native-websocket/wicket-native-websocket-javax/src/main/java/org/apache/wicket/protocol/ws/javax/JavaxWebSocketProcessor.java

```diff
@@ -17,15 +17,19 @@
 package org.apache.wicket.protocol.ws.javax;
 
 import java.nio.ByteBuffer;
+import java.util.List;
 
 import javax.websocket.EndpointConfig;
 import javax.websocket.MessageHandler;
+import javax.websocket.PongMessage;
```

Comment:
I still don't understand how JSR 356 spec makes use of javax.websocket.PongMessage

Reply:
Server pings client --> client implementation answers with a pong --> specification does not force client to offer a onping, which is  BAD, because we can't use it for reconnect timeout thing :-(. At least this is what I understood after googling around